catroid/src/main/java/org/catrobat/catroid/ProjectManager.java

```diff
@@ -243,6 +243,14 @@ public void loadProject(String projectName, Context context) throws LoadingProje
 				project.setCatrobatLanguageVersion(0.992f);
 			}
 			if (project.getCatrobatLanguageVersion() == 0.992f) {
+				project.updateCollisionFormulasToNewVersion();
+				project.setCatrobatLanguageVersion(0.993f);
+				asynchronousTask = false;
+				saveProject(context);
+				loadProject(projectName, context);
```

Comment:
This way, one can be absolutely sure that everthing was changed properly. Furthermore, the new program has to be saved anyways. As this has to be done only once, in my mind the performance loss is bearable